src/interrogate/visit.py

```diff
@@ -1,11 +1,19 @@
-# Copyright 2020 Lynn Root
+# Copyright 2020-2024 Lynn Root
 """AST traversal for finding docstrings."""
+from __future__ import annotations
 
 import ast
 import os
 
 import attr
 
+from interrogate.config import InterrogateConfig
+
+
+DocumentableFunc = ast.AsyncFunctionDef | ast.FunctionDef
+DocumentableFuncOrClass = DocumentableFunc | ast.ClassDef
+DocumentableNode = DocumentableFuncOrClass | ast.Module
```

Comment:
same as above

Reply:
Thanks so much for your review @trag1c - I am so behind on keeping up with the changes in typing 🙈